src/index.html

```diff
@@ -10,8 +10,175 @@
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+            Surname:
+          <input
+            type="text"
+            autocomplete="off"
+            name="surname"
+            required
+            >
+        </label>
+
+        <label class="form-field">
+            Name:
+          <input
+            type="text"
+            autocomplete="off"
+            name="name"
+            required
+            >
+        </label>
+
+        <label class="form-field">
+            How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            >
+        </label>
+
+        <label class="form-field">
+            Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            >
+        </label>
+
+        <label class="form-field last">
+            I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            >
+        </label>
+
+      </fieldset>
+
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="form-field">
+            E-mail:
+          <input
+            type="email"
+            name="emailInput"
+            placeholder="email@example.com"
+            >
+        </label>
+
+        <label class="form-field last">
+            Password:
+          <input
+            type="password"
+            name="passwordInput"
+            minlength="5"
+            maxlength="20">
+        </label>
+
+      </fieldset>
+
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+            Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              name="answer"
+              value="yes"
+              >
+              Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="answer"
+              value="no"
+              >
+              No
+          </label>
+        </div>
+
+        <label class="form-field">
+              What is your favorite color?
+          <input
+              type="color"
+              name="colorInput"
+              >
+        </label>
+
+        <label class="form-field">
+              What time do you go to bed?
+          <input
+              type="time"
+              step="1"
+              name="bedTime"
+              >
+        </label>
+
+        <label class="form-field">
+              What are your favorite brand of cars?
+          <select name="favColor" multiple>
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
```

Comment:
you're missing a `value` attribute for options - values won't be passed without it
```suggestion
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
```